util/reggen/Makefile

```diff
@@ -7,7 +7,7 @@ all: lint
 
 # We need a directory to build stuff and use the "util/reggen" namespace
 # in the top-level build-bin directory.
-repo-top := ../../../..
+repo-top := ../..
```

Comment:
i have a really dumb question.
How come changing the path here does not break anything..? was the original build-dir wrong..?

Reply:
Since, it builds the directory somewhere upper directories (if I understand correctly). Then it won't fail the flow.

Reply:
Yeah, I just checked to confirm and the original build-dir is just wrong. Correcting it could change the behaviour though (since the current behaviour is that it is built in a totally separate directory that nothing else touches).

@drewmacrae do you have any thoughts on the use of build-bin/util/reggen as a build directory here? Would that conflict with anything bazel does?

Reply:
I suspect somehow OTBN symlink reggen under `hw/ip/otbn/dv/` or something like that. Can any OTBN contributors confirm?

Reply:
@andreaskurth @ctopal @vogelpi

Reply:
OTBN register RTL can be generated with `util/regtool.py -r -t hw/ip/otbn/rtl hw/ip/otbn/data/otbn.hjson`. This still works after this PR has been merged. Does this answer this concern?